README.md

```diff
@@ -246,6 +246,39 @@ asks the user to confirm their deletion before submitting the actual deletion re
 This page is not technically part of the subflow and as such, does not need to be denoted
 with `subflow: subflowName` in the `flows-config.yaml`.
 
+### Subflows Data
+Subflow information will be saved in your applications database within the larger JSON `inputData` as an array of subflow iterations
+like this example of a household subflow with two iterations:
+```JSON
+  "household": [
+    {
+      "uuid": "e2c18dfe-98e9-430f-9a4f-3511966d3128",
+      "iterationIsComplete": true,
+      "householdMemberLastName": "Example",
+      "householdMemberFirstName": "Person",
+      "householdMemberRelationship": "Spouse",
+      "householdMemberRecentlyMovedToUS": "No"
+    },
+    {
+      "uuid": "3fafdb80-f7e7-4fdc-aefd-461e0b6c1cdf",
+      "iterationIsComplete": false,
+      "householdMemberLastName": "Other Example",
+      "householdMemberFirstName": "Person",
+      "householdMemberRelationship": "Child",
+      "householdMemberRecentlyMovedToUS": "No"
+    }
+  ],
+```
+Note that all information submitted for a single loop (iteration) through your subflow will show up within a single array index.
+The index includes a `uuid` field, which is a unique identifier for that iteration within the subflow. 
+
+#### Completed iterations
```

Comment:
Are we capturing that when the last screen is a GET requests out of a subflow,  iterationIsComplete is set to true?

Reply:
Made a small update to clarify